lib/commands/sdk.js

```diff
@@ -535,713 +473,575 @@ async function select(logger, config, cli) {
  * @param {Function} finished - Callback when the command finishes
  */
 SdkSubcommands.install = {
-	conf: function (_logger, _config, _cli) {
+	conf(_logger, _config, _cli) {
 		return {
 			// command examples:
 			// ti sdk install
 			// ti sdk install --default
+			// ti sdk install latest
+			// ti sdk install latest-rc
+			// ti sdk install latest-beta
 			// ti sdk install something.zip
 			// ti sdk install something.zip --default
 			// ti sdk install http://builds.appcelerator.com/mobile/master/mobilesdk-3.2.0.v20130612114042-osx.zip
 			// ti sdk install 3.1.0.GA
 			// ti sdk install 3.1.0.GA --default
-			desc: __('download the latest Titanium SDK or a specific version'),
+			// ti sdk install master:11.1.0.v20220614174006
+			desc: 'download the latest Titanium SDK or a specific version',
 			args: [
 				{
-					desc: __('the version to install, "latest", URL, or zip file'),
+					desc: 'the version to install, "latest", URL, zip file, or <branch>:<build_name>',
 					name: 'version',
 					required: true
 				}
 			],
 			flags: {
 				default: {
 					abbr: 'd',
-					desc: __('set as default SDK'),
+					desc: 'set as default SDK'
 				},
 				force: {
 					abbr: 'f',
-					desc: __('force re-install')
+					desc: 'force re-install'
 				},
 				'keep-files': {
 					abbr: 'k',
-					desc: __('keep downloaded files after install'),
+					desc: 'keep downloaded files after install'
+				}
+			},
+			options: {
+				branch: {
+					abbr: 'b',
+					desc: 'the branch to install from or "latest" (stable)',
+					hint: 'branch name'
 				}
 			}
-			// options: {
-			// 	branch: {
-			// 		abbr: 'b',
-			// 		desc: __('the branch to install from or "latest" (stable)'),
-			// 		hint: __('branch name')
-			// 	}
-			// }
 		};
 	},
 	fn: callbackify(install)
 };
 
+/**
+ * A regex to test if a string is a URL or path to a zip file.
+ * @type {RegExp}
+ */
+const uriRegExp = /^(https?:\/\/.+)|(?:file:\/\/(.+))$/;
+
 /**
  * @param {Object} logger - The logger instance
  * @param {Object} config - The CLI config object
  * @param {CLI} cli - The CLI instance
  */
 async function install(logger, config, cli) {
-	const installLocation = afs.resolvePath(cli.env.installPath);
-
-	// make sure the install location exists
-	try {
-		await ensureInstallLocation(installLocation);
-	} catch (error) {
-		logger.error(error);
-		throw error;
+	const titaniumDir = afs.resolvePath(cli.env.installPath);
```

Comment:
You've spaced the others nicely, do that here too if you're wanting to do that?

Reply:
Fixed.